src/components/CheckboxGroup/index.tsx

```diff
@@ -0,0 +1,48 @@
+import { APP_CATEGORY } from '@/constants/discover';
+import { useState } from 'react';
+
+export interface ICheckboxOption {
+  value: APP_CATEGORY;
+  label: string;
+}
```

Comment:
if you import directly then you do not need to export here and this should be a type since it is not use by the component

Reply:
Updated